### ISSUE TITLE
Ability to wrap element for complex form rendering

### ISSUE THREAD

**Issue by ntbatyr** [laminas/laminas-form#216]
### Feature Request

Adding a parent element, or at least wrapping it in a single block, made it easier to work with forms in complex rendering, without passing at a time. Usage practice can be dynamically assembled elements, the volume of which would not be small; It would also make it possible to write less code in the `phtml` file.

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | no
| BC Break    | no

#### Summary

`laminas-form/src/View/Helper/FormRow::render` method

Implementation options: `wrap_template` option
- request a template for `sprintf`
- request a callable function as an argument already pass `markup`
- request an array of options with attributes and element tag (although in this case only one element will be generated)

**Comment by froschdesign** [comment]
I think this request is a duplication of #62.

**Comment by ntbatyr** [comment]
> I think this request is a duplication of #62.

Yes, I didn't notice. Sorry! But so many times have been passed.

**Comment by froschdesign** [comment]
@ntbatyr 
> But so many times have been passed.

Create a pull request and the wait is over. 👍🏻 

_Any help is welcome and we are thankful for any contribution!_

**Comment by mimmi20** [comment]
@ntbatyr Did you try this?
```php
$this->formRow()->setPartial('the Partial');
$this->formRow($element);
...
```
